variables.tf

```diff
@@ -152,32 +152,36 @@ variable "administrative_trigger_policy_enabled" {
   default     = true
 }
 
-variable "trigger_retries_policy_enabled" {
-  type        = bool
-  description = "Flag to enable/disable the stack automatic retries trigger policy"
-  default     = false
-}
-
-variable "trigger_dependency_policy_enabled" {
-  type        = bool
-  description = "Flag to enable/disable the stack trigger dependency policy"
-  default     = true
-}
-
-variable "access_policy_enabled" {
-  type        = bool
-  description = "Flag to enable/disable the stack access policy"
-  default     = true
-}
-
-variable "push_policy_enabled" {
-  type        = bool
-  description = "Flag to enable/disable the stack push policy"
-  default     = true
+variable "policies_create" {
+  type        = list(string)
+  description = "List of default policies to create in Spacelift (these policies will not be attached to Spacelift stacks by default, use `var.policies_attach`)"
+  default = [
+    "access.default",
+    "git_push.default",
+    "plan.default",
+    "trigger.dependencies",
+    "trigger.retries"
+  ]
+}
+
+variable "policies_attach" {
+  type        = list(string)
+  description = "List of default policies to attach to all Spacelift stacks"
+  default = [
+    "git_push.default",
+    "plan.default",
+    "trigger.dependencies"
+  ]
+}
+
+variable "policies_path" {
+  type        = string
+  description = "Path to the catalog of default policies"
+  default     = "catalog/policies"
 }
 
-variable "plan_policy_enabled" {
-  type        = bool
-  description = "Flag to enable/disable the stack plan policy"
-  default     = true
+variable "additional_policy_ids" {
```

Comment:
```suggestion
variable "additional_policies_enabled" {
```

Reply:
I think `additional_policy_ids` better convey the meaning: the caller creates additional policies in their component and provides their IDs to attach to each Spacelift stack (they are enabled by definition)